src/main/java/core/basesyntax/service/fileservice/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.fileservice;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> data = new ArrayList<>();
+        try (BufferedReader fileReader = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = fileReader.readLine()) != null) {
+                data.add(line);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file", e);
+        }
+        return data;
```

Comment:
```suggestion
        try {
            return Files.readAllLines(Path.of(filePath));
        } catch (IOException e) {
            throw new RuntimeException(FILE_EXCEPTION_MESSAGE
                    + "\nFile, " + filePath, e);
        }
```
This can be simplified. And let's create a constant for the error message.

Reply:
our code is doing the same things, we work with small data, so I guess io is okay there

Reply:
we work with small data => it's always starts with the small data, then code has a tendency to grow. readAllLines suites better, cause it will shorten the code